demo/index.ts

```diff
@@ -85,23 +94,28 @@ const {
   additionalRoleOptions,
 });
 
-const { plugin: elementPlugin, insertElement, nodeSpec } = buildElementPlugin({
+const {
+  plugin: elementPlugin,
+  insertElement,
+  nodeSpec,
+  getElementDataFromNode,
+} = buildElementPlugin({
   "demo-image-element": createDemoImageElement(onSelectImage, onDemoCropImage),
-  imageElement,
-  embedElement: createEmbedElement({
+  image: imageElement,
+  embed: createEmbedElement({
     checkThirdPartyTracking: mockThirdPartyTracking,
     convertTwitter: (src) => console.log(`Add Twitter embed with src: ${src}`),
     convertYouTube: (src) => console.log(`Add youtube embed with src: ${src}`),
     createCaptionPlugins,
   }),
-  interactiveElement: createInteractiveElement({
+  interactive: createInteractiveElement({
     checkThirdPartyTracking: mockThirdPartyTracking,
     createCaptionPlugins,
   }),
-  codeElement,
-  pullquoteElement,
-  richlinkElement,
-  videoElement: createVideoElement({
+  code: codeElement,
+  pullquote: pullquoteElement,
+  "rich-link": richlinkElement,
```

Comment:
Interested in why you've gone for kebab case when it creates this necessity?

Reply:
Good question! It's because this matches our requirements in Composer. Before #156, this small change broke our plugin, so it's nice to have it in here for testing purposes. Could add a comment to that effect!

Reply:
(There are also automated tests for this)